Userland/Libraries/LibCore/ConfigFile.cpp

```diff
@@ -76,6 +76,12 @@ void ConfigFile::reparse()
 
     while (m_file->can_read_line()) {
         auto line = m_file->read_line();
+
+        if (line.is_null()) {
```

Comment:
The commit message mentions "null lines"
What exactly is a "null line"?

Reply:
Core::File::read_line() sometimes returns a null string (e.g. when there is no line to read, like in binary files, and it seems that not only if can_read_line() is false).
"Null lines" are these lines.